### DISCUSSION TITLE
electron-vite打包时，在哪里可以配置安装后写入的注册表信息呢

### DISCUSSION THREAD

**Discussion by blanc1412** [alex8088/electron-vite#534] [General]
我想在安装项目时，写入注册表信息，配置一个protocol。在项目中什么地方可以配置呢。

**Comment by alex8088** [comment]
electron本身支持protocol，自行查找文档。如果是代码操作注册表可结合第三方库，如果安装时配置可看electron-builder的相关文档了解

**Reply by blanc1412** [reply]
谢谢~

**Reply by blanc1412** [reply]
e-v是在electron-builder.yml文件中配置吗，只看到这个文件
